src/library-authoring/data/api.js

```diff
@@ -22,3 +27,12 @@ export async function getContentLibrary(libraryId) {
   const { data } = await getAuthenticatedHttpClient().get(getContentLibraryApiUrl(libraryId));
   return camelCaseObject(data);
 }
+
+export async function getLibraryBlockTypes(libraryId) {
+  if (!libraryId) {
+    throw new Error('libraryId is required');
+  }
+
+  const { data } = await getAuthenticatedHttpClient().get(getLibraryBlockTypesUrl(libraryId));
+  return camelCaseObject(data);
+}
```

Comment:
I think it is worth to explicitly type the output here

Reply:
Updated: https://github.com/open-craft/frontend-app-course-authoring/pull/41/commits/499acedcf0ddeed9c2b75aeb2c164a55d992e8e4